cupy/cuda/function.pyx

```diff
@@ -144,7 +144,7 @@ cdef inline size_t _get_stream(stream) except *:
 cdef _launch(intptr_t func, Py_ssize_t grid0, int grid1, int grid2,
              Py_ssize_t block0, int block1, int block2,
              args, Py_ssize_t shared_mem, size_t stream,
-             bint enable_cooperative_groups=False):
+             bint enable_cooperative_groups=False, str log_stream=None):
```

Comment:
I don't think you need to touch this file, because by the time `launch` is called, the compilation is already done, and there's no place where `log_stream` can be used.
```suggestion
             bint enable_cooperative_groups=False):
```